src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -1,7 +1,5 @@
 <?php
-
```

Comment:
Uuuuh… why did you remove all these blank lines?

Reply:
I sorry but phpstorm reformat code before commit, I'll revert to last changes